### PR TITLE
GDB-10641-some-minor-improvements-in-tests

### PR DESCRIPTION
**Pull request by svilenvelikov**
## What
Refactored and extended some acceptance tests in shared components package.

## Why
To improve the code coverage and test package structure. 

## How
Separated tests, steps and test pages for each component.
Implemented tests for the dropdown menu.

## Checklist
- [x] Branch name
- [x] Target branch
- [x] Commit messages
- [x] Squash commits
- [x] MR name
- [x] MR Description
- [x] Tests

### CODE REVIEW

File: packages/shared-components/src/pages/dropdown/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <script src="./main.js" defer></script>
+  <link rel="stylesheet" href="../css/bootstrap.min.css">
+  <link rel="stylesheet" href="../css/font-awesome.min.css">
+</head>
+<body>
+<div style="display: flex; flex-direction: row">
+  <onto-dropdown id="dropdown1" dropdown-button-name="Dropdown 1"
+                 dropdown-button-tooltip="dropdown 1 tooltip"></onto-dropdown>
+
+  <onto-dropdown id="dropdown2" dropdown-button-name="Dropdown 2" dropdown-alignment="0"></onto-dropdown>
```
Comment (boyan-tonchev) [inline packages/shared-components/src/pages/dropdown/index.html:27]: Sorry about this comment (I know it's my fault) but could you change the DropdownItemAlignment to:
```
export enum DropdownItemAlignment {  
  RIGHT = 'right',  
  LEFT = 'left'  
}
```
This way, we can configure the dropdown more meaningfully using `dropdown-alignment="left"` or `dropdown-alignment="right`.

File: packages/shared-components/src/components/onto-language-selector/onto-language-selector.tsx [hunk 2]
```diff
@@ -14,6 +14,12 @@ export class OntoLanguageSelector {
   private items: DropdownItem<string>[] = [];
   private readonly onLanguageChangeSubscription: () => void;
 
+  /**
+   * Specifies the dropdown items' alignment. If not provided, the items and the dropdown button will be aligned to the left.
+   *
+   */
+  @Prop() dropdownAlignment: DropdownItemAlignment = DropdownItemAlignment.LEFT;
```
Comment (boyan-tonchev) [inline packages/shared-components/src/components/onto-language-selector/onto-language-selector.tsx:21]: This has to be set to `DropdownItemAlignment.RIGHT` or configured within the header component. Currently, a horizontal scroll appears when the language selector is opened.
image

**Review by boyan-tonchev — APPROVED** [review]

File: packages/shared-components/src/pages/dropdown/index.html (hunk 1 above)
Comment (svilenvelikov) [reply packages/shared-components/src/pages/dropdown/index.html]: changed

File: packages/shared-components/src/components/onto-language-selector/onto-language-selector.tsx (hunk 2 above)
Comment (svilenvelikov) [reply packages/shared-components/src/components/onto-language-selector/onto-language-selector.tsx:21]: fixed

**Pull request merged**